### DISCUSSION TITLE
Parity with Desktop App and Vulnerability Patches

### DISCUSSION THREAD

**Discussion by riebs333** [dlemstra/Magick.NET#1469] [General] [answered]
How frequently do updates to Magick.net follow changes to the ImageMagic desktop app?  I am considering switching to use Magick.net to replace C# code that currently calls out to the exe and curious on how quickly features or fixes follow to Magick.net or if that is even a concern.  If a vulnerability were found in the Magick.net package, how quickly are those addressed?

**Comment by dlemstra** [comment, marked as answer]
I am also a maintainer of the ImageMagick project and as you can see in the releases the version of ImageMagick is very recent: https://github.com/dlemstra/Magick.NET/releases. And I try to do a release once every 1 or 2 months. But if you have something that you want quicker you are free to ask if I can do a release. But I cannot guarantee I have time to do that.

Vulnerability are solved as soon as possible but don't forget I am the only person in this project that publishes releases and this is an open source project that comes with no guarantees. If you sponsor this project you will get a message when a vulnerability is found in this project or one of it's dependencies.
